### PR TITLE
Load external, builtin & local apps in parallel

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/ethcore/parity/issues/3339

### CODE REVIEW

**Comment by jacogr** [comment]
I asked @chevdor to test, he reports that loading times are now back to normal.

File: js/src/views/Dapps/dappsStore.js [hunk 1]
```diff
@@ -210,39 +202,43 @@ export default class DappsStore {
...
+        this.externalApps = JSON.parse(stored);
+      } catch (error) {
+        console.warn('DappsStore:readExternalApps', error);
+      }
+    }
+  }
+
+  _writeExternalApps () {
+    try {
+      localStorage.setItem(LS_KEY_EXTERNAL, JSON.stringify(this.externalApps));
```
Comment (ngotchac) [inline js/src/views/Dapps/dappsStore.js]: Why not using the `store` package that is installed ?

Comment (jacogr) [inline js/src/views/Dapps/dappsStore.js]: We actually have an issue logged to rework the localStorage - I prefer to do it properly everywhere and set the standard to doing it piecemeal and forgetting about the rest. (So basically when creating the store didn't re-invent the code that was there.)

**Comment by gavofyork** [comment]
gitlab fails js tests

**Comment by jacogr** [comment]
Master merged in, tests now updated & passing.

**Pull request merged**